pkg/Microsoft.Private.CoreFx.UAP/uap.rids.props

```diff
@@ -1,22 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <_aotSuffix Condition="'$(TargetGroup)' == 'uapaot'">-aot</_aotSuffix>
+    <PackageRID>$(TargetRid)</PackageRID>
```

Comment:
I still see PackageRID used in multiple places where it is just set directly to TargetRID.  A few points: why do the rename? are you sure you every place that needs this remapping? why not just clean up PackageRID entirely?